dev/Materials/Acrylic/TestUI/AcrylicColorPage.xaml.cs

```diff
@@ -67,7 +67,8 @@ public static IEnumerable<string> AcrylicBrushNames
                     "SystemControlAltMediumLowAcrylicWindowMediumBrush",
                     "SystemControlAltMediumLowAcrylicElementMediumBrush",
                     "SystemControlAltHighAcrylicWindowBrush",
-                    "SystemControlAltHighAcrylicElementBrush" };
+                    "SystemControlAltHighAcrylicElementBrush"
+                };
```

Comment:
What caused this test to fail ?

Reply:
There is a test that uses a regex to validate the brushes names. The new brushes don't adhere to that pattern (e.g. don't start with `SystemControl`) so it failed the test. Not sure if keeping that test now makes sense though.